storage/spanner.go

```diff
@@ -828,7 +833,7 @@ func parseRow(r *spanner.Row, colDDL map[string]string) (map[string]interface{},
 				}
 				singleRow[k] = m
 			}
-		case "INT64":
+		case "N":
```

Comment:
Have u tested the scenarios for INT64, FLOAT64, NUMERIC as all will be N on dynamo